frontend/packages/dev-console/integration-tests/OWNERS

```diff
@@ -0,0 +1,23 @@
+reviewers:
+  - abhi-kn
+  - andrewballantyne
+  - abhinandan13jan
+  - debsmita1
+  - divyanshiGupta
+  - christianvogt
+  - invincibleJai
+  - jerolimov
+  - karthikjeeyar
+  - nemesis09
+  - rohitkrai03
+  - rottencandy
+  - sahil143
+  - vikram-raj
+approvers:
+  - andrewballantyne
+  - christianvogt
+  - invincibleJai
+  - rohitkrai03
+  - makambalaji
+labels:
+  - component/dev-console
```

Comment:
I believe you won't need a label here because the parent OWNERS file will add the `dev-console` label.